packages/api/controllers/operatorServices/event/consumeEvent.js

```diff
@@ -22,34 +20,36 @@ module.exports = async function consumeEvent() {
   }
 };
 
-function processEvents(eventRequest) {
+async function processEvents(eventRequest) {
   const response = JSON.parse(eventRequest.data);
-  const currentTime = new Date();
   const eventBody = new event({
     id: uuid(),
     eventId: response?.uuid,
     propertyName: response?.websiteName || "NA",
     spaName: response?.spaName || "NA",
     version: 1,
-    env: response.environmentName,
+    env: response?.environmentName || "NA",
     branch: "main",
-    path: "",
-    url: "",
+    state: response.state,
+    path: response?.contextPath || "NA",
+    accessUrl: getUrl(response.accessUrl),
     code: getCode(response.state),
     failure: false,
     isActive: true,
-    createdAt: currentTime,
-    updatedAt: currentTime,
     traceId: response?.uuid,
   });
   console.log(eventBody);
-  Promise.all([createEventRequest(eventBody), createEventTimeTraceRequest(response)]);
+  await createEventRequest(eventBody);
+  await createEventTimeTraceRequest(response);
 }
 
 function getCode(state) {
-  if (state == "mapping file loaded into memory") return "WEBSITE_CREATE_STARTED";
-  if (state == "spa deployment ops performed") return "WEBSITE_CREATE";
-  return "";
+  if (state == "spa deployment ops performed") return "WEBSITE_CREATETED";

```

Comment:
@soumyadip007 because you are using these strings multiple times can you please put these under a property file or create an [enum](https://www.w3schools.com/typescript/typescript_enums.php) for the same

Reply:
will resolve

Reply:
resolved